### PR TITLE
Bring CHANGELOG up to date

### PR DESCRIPTION
**Pull request by VSPPedro**
:tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+[#1406]: https://github.com/thoughtbot/shoulda-matchers/pull/1406
+[#1418]: https://github.com/thoughtbot/shoulda-matchers/pull/1418
+[#1419]: https://github.com/thoughtbot/shoulda-matchers/pull/1419
+[#1424]: https://github.com/thoughtbot/shoulda-matchers/pull/1424
+[#1427]: https://github.com/thoughtbot/shoulda-matchers/pull/1427
+
+### Backward-incompatible changes
+
+* Drop support for Rails 4.2, 5.0 and 5.1 as well as Ruby 2.4 and 2.5
+  they've been end-of-lifed. The gem now supports Ruby 2.6+ and Rails 5.2+.([#1412], [#1415], [#1422], [#1428],[#1429])
```
Comment (mcmire) [inline CHANGELOG.md:31]: Should you have a space after the period?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+### Bug fixes
+
+* Replace in?, method from ActiveSupport, with include? to prevent exception "undefined method `in?`" on non-Rails app. ([#1405])
+
+[#1405]: https://github.com/thoughtbot/shoulda-matchers/pull/1405
+
+### Features
+
+* Add support for Rails 6.1. No new Rails 6.1 features are supported, but only
+  existing features that broke with the upgrade.([#1418])
```
Comment (mcmire) [inline CHANGELOG.md:14]: Should you have a space after the period?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## Unreleased
+
+### Bug fixes
+
+* Replace in?, method from ActiveSupport, with include? to prevent exception "undefined method `in?`" on non-Rails app. ([#1405])
```
Comment (mcmire) [inline CHANGELOG.md:7]: Maybe place backticks around `in?` and `include?` here?

**Review by mcmire — APPROVED** [review]
A few suggestions, but this looks good to me. Thanks for doing this!

**Comment by VSPPedro** [comment]
Oops, done!

**Pull request merged**